tests/Tests/Vector/Property.hs

```diff
@@ -541,8 +541,8 @@ testOrdFunctions _ = $(testProperties
 listMaxIndexFMW :: Ord a => [a] -> Int
 listMaxIndexFMW  = ( fst  . extractFMW .  sconcat . DLE.fromList . fmap FMW . zip [0 :: Int ..])
 
-listMaxIndexLMW :: Ord a => [a] -> Int
-listMaxIndexLMW = ( fst  . extractLMW .  sconcat . DLE.fromList . fmap LMW . zip [0 :: Int ..])
+-- listMaxIndexLMW :: Ord a => [a] -> Int
+-- listMaxIndexLMW = ( fst  . extractLMW .  sconcat . DLE.fromList . fmap LMW . zip [0 :: Int ..])
```

Comment:
Why is it left commented out?

Reply:
Not used any more since we don't have function that picks last tie. I commented in out to keep it around. Just in case

Reply:
I think its best to remove dead code like this, it can be later recovered easily from `listMaxIndexFMW` if necessary.

Reply:
Removed